src/sinks/humio_logs.rs

```diff
@@ -287,7 +345,7 @@ mutation {{
         humio_type: String,
 
         #[serde(rename = "@error")]
-        error: Option<bool>,
+        error: Option<String>,
```

Comment:
Realized it is actually sending back `"true"` rather than `true` when errors occur.

Reply:
Good catch.